### DISCUSSION TITLE
文本单元格能够实现自动换行的同时，调整行高

### DISCUSSION THREAD

**Discussion by Lee94** [antvis/S2#2911] [Q&A] [answered]
现在的自动换行还是会出现省略号，不能自动调整行高

**Comment by KzqKzq** [comment, marked as answer]
我也有这样的需求，v2的版本目前看是因为有maxLines设置的作用限制，可以看[类似的问题](https://github.com/antvis/S2/issues/2594)，目前的临时方案可以考虑给maxLines设置一个非常大的值
![image](https://github.com/user-attachments/assets/67531a10-3b6e-4376-9f8a-397ef41d3184)
